### DISCUSSION TITLE
to-do list block : custom text properties for checked item

### DISCUSSION THREAD

**Discussion by LaurentAtJoes** [TriliumNext/Notes#699] [General]
Hi all,
I'm trying to add a bit of custom css to the default theme. Disclaimer: I'm no CSS expert.

I can't figure out how to set custom color and text-decoration _only for CHECKED ITEMS_ .

If I try out this css code, it obviously sets those text properties for _all_ items. (the part for the checked box background works well)
```
.ck-editor__editable.ck-content .todo-list .todo-list__label>span[contenteditable=false]>input[checked]:before {
    background: #666666 !important;
    border-color: #222222 !important;
}
.ck-content .todo-list .todo-list__label .todo-list__label__description {
    color: #c9c9c9 !important;
    text-decoration: line-through !important;
}
```

I assume there maybe a way similar to what is used for the _checkbox itself_ (when checked), with the `>span[contenteditable=false]>input[checked]:before` part, but I have no idea how to write that.

This is the desired result : (here with manual text formatting, for the example)
![image](https://github.com/user-attachments/assets/343bd6dc-ee79-4410-814f-7c44123f8589)

Any help would be appreciated

**Comment by SiriusXT** [comment]
It can automatically apply a strikethrough to the text following a checked checkbox.
```
.todo-list__label:has(input:checked) .todo-list__label__description {
    text-decoration: line-through;
    opacity: 0.6;
}
```

**Comment by LaurentAtJoes** [comment]
Yes, this works perfectly fine, many thanks.
